### PR TITLE
docs: Extend documentation on allowed-gadgets.

### PR DESCRIPTION
**Pull request by eiffel-fl**

### CODE REVIEW

File: docs/reference/restricting-gadgets.mdx [hunk 1]
```diff
@@ -7,8 +7,17 @@ description: How to restrict the gadgets that can be run
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-It is possible to limit the gadgets that can be run by using the
-`--allowed-gadgets` flag. By default, all gadgets are allowed.
+It is possible to limit the gadgets that can be run by using the `--allowed-gadgets` flag.
+With this flag, you can restrict running the gadgets using the following:
+1. By digest,
+2. By tag
+3. Or by using a prefix, like `ghcr.io/inspektor-gadget/gadget/*.
```
Comment (flyth) [inline docs/reference/restricting-gadgets.mdx:14]:
```suggestion
3. Or by using a prefix with a wildcard at the end, like `ghcr.io/inspektor-gadget/gadget/*`.
```

File: docs/reference/restricting-gadgets.mdx [hunk 2]
```diff
@@ -7,8 +7,17 @@ description: How to restrict the gadgets that can be run
...
+
+By default, all gadgets are allowed.
```
Comment (flyth) [inline docs/reference/restricting-gadgets.mdx:16]:
```suggestion
By default, all gadgets are allowed, although other restrictions (like a signature check) could still keep it from running.
```

File: docs/reference/restricting-gadgets.mdx [hunk 3]
```diff
@@ -75,3 +84,154 @@ RUNTIME.CONTAINERN… TIMESTAMP  PID        UID        GID        MNTNS_ID   ERR
 ```
 </TabItem>
 </Tabs>
+
+### By tag
+
+We also offer the posibility to restrict running by gadget tags:
```
Comment (flyth) [inline docs/reference/restricting-gadgets.mdx:90]:
```suggestion
We also offer the possibility to restrict running by image tags:
```
Unsure about this one.

**Review by flyth — COMMENTED** [review]
Thanks!

Not in this PR, but: do we want to extend wildcard usage to also make things like `ghcr.io/inspektor-gadget/gadget/*_prod` possible? Or maybe even accept regular expressions?

File: docs/reference/restricting-gadgets.mdx [hunk 4]
```diff
@@ -7,8 +7,17 @@ description: How to restrict the gadgets that can be run
...
 
-It is possible to limit the gadgets that can be run by using the
-`--allowed-gadgets` flag. By default, all gadgets are allowed.
+It is possible to limit the gadgets that can be run by using the `--allowed-gadgets` flag.
+With this flag, you can restrict running the gadgets using the following:
+1. By digest,
+2. By tag
+3. Or by using a prefix, with a wildcard at the end, like `ghcr.io/inspektor-gadget/gadget/*`.
+
+By default, all gadgets are allowed, although other restrictions (like a signature check) could still keep it from running.
```
Comment (mauriciovasquezbernal) [inline docs/reference/restricting-gadgets.mdx:16]:
```suggestion
By default, all gadgets are allowed, although other restrictions (like a signature check) could still keep them from running.
```

?

**Pull request merged**